5.7/root/usr/share/container-scripts/mysql/README.md

```diff
@@ -187,16 +188,25 @@ location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by settin
 
 Extending image
 ---------------
-This image can be extended using [source-to-image](https://github.com/openshift/source-to-image).
+This image can be extended in Openshift using the `Source` build strategy or via the standalone
+[source-to-image](https://github.com/openshift/source-to-image) application (where available).
+For this, we will assume that you are using the `rhscl/mysql-57-rhel7` image,
+available via `mysql:5.7` imagestream tag in Openshift.
 
-For example, to build a customized MariaDB database image `my-mysql-rhel7`
-with a configuration in `~/image-configuration/` run:
+For example, to build a customized MySQL database image `my-mysql-rhel7`
+with a configuration in `image-configuration/` run:
 
 ```
-$ s2i build ~/image-configuration/ rhscl/mysql-57-rhel7 my-mysql-rhel7
+$ oc new-app mysql:5.7~./image-configuration/ --name my-mysql-rhel7
```

Comment:
I don't think local dirs work in OpenShift, what about to use:
```
oc new-app mysql:5.7~https://github.com/yourorg/my-own-mysql-configuration.git --name my-mysql-rhel7
```
or even
```
oc new-app mysql:5.7~https://github.com/sclorg/mysql-container.git --context-dir=examples/extend-image --name my-mysql-rhel7
```

Reply:
Ah right, missed that it is not explicitly stated that the `image-configuration` dir must be an accessible git repository containing the configuration. Thanks for spotting this!

I will change this to the longer example.